src/applications/my-education-benefits/components/confirmation/ConfirmationApproved.jsx

```diff
@@ -20,14 +20,14 @@ export default function ConfirmationApproved({
           of Eligibility is now available. A physical copy will also be mailed
           to your mailing address.
         </p>
-        <a
-          type="button"
-          className="usa-button-primary va-button-primary"
-          href={LETTER_URL}
+        <va-link
           download
-        >
-          Download your Certificate of Eligibility
-        </a>
+          href={LETTER_URL}
+          text="Download your Certificate of Eligibility"
+          class="vads-u-padding-bottom--2"
+        />
```

Comment:
Reference: https://design.va.gov/components/link/#when-to-use-a-download-link

>  When to use a Download link - Downloading files. Use for download links including but not limited to PDFs and Excel files.

We have a download variation of `va-link` available the anchor styled to look like a button has been replaced with that:

**before**
![Screenshot 2024-01-04 at 2 17 13 PM](https://github.com/department-of-veterans-affairs/vets-website/assets/872479/39cd37aa-60f3-4627-9904-7cc4c4effa77)

**after**
![Screenshot 2024-01-04 at 2 22 48 PM](https://github.com/department-of-veterans-affairs/vets-website/assets/872479/ad986a2c-4842-4d90-90af-58c291f51e49)

Reply:
Looks good!